.github/workflows/test.yml.template

```diff
@@ -0,0 +1,6 @@
+on: [push, pull_request]
+name: Continuous integration
+jobs:
+  test:
+    name: run tests and linters
+    uses: kubewarden/github-actions/.github/workflows/reusable-test-policy-rego.yml@v1
```

Comment:
I just noticed that in the [swift template](https://github.com/kubewarden/swift-policy-template/pull/2/files) you named the file as `test.yml`. Is there a reason why is different here?

Reply:
oh shoot. They should have been templates on the swift ones, too. I will open a PR for that. Thanks!

Reply:
Done in https://github.com/kubewarden/swift-policy-template/pull/3.

Reply:
:)